react/mod.ts

```diff
@@ -59,6 +60,16 @@ export default function (userOptions?: Partial<Options>) {
           script.setAttribute("type", "module");
           script.setAttribute("src", mainUrl.replace(".tsx", ".js"));
           page.document!.body.appendChild(script);
+
+          if (hydrate) {
+            const appPath = "./" + basename(site.root()) + src; // TOOD: fix path when releasing plugin
```

Comment:
Lume has the function `site.src()` to return a path for any file in the src folder.
So `site.src(src)` returns the full path of a file inside `src` folder. Then you have to convert it to a file url so it can be imported dynamically:

```js
const appPath = toFileUrl(site.src(src)).href;
```